ephios/core/models/users.py

```diff
@@ -158,18 +159,16 @@ def get_workhour_items(self):
                     (
                         F("shift__end_time") - F("shift__start_time")
                     ),  # calculate length of shift in μs
-                    output_field=models.IntegerField(),
-                )
-                / 1000000
-                / 3600,  # convert microseconds to seconds to hours
+                    output_field=models.DurationField(),
+                ),
                 date=ExpressionWrapper(TruncDate(F("shift__start_time")), output_field=DateField()),
                 reason=F("shift__event__title"),
             )
             .values("hours", "date", "reason")
         )
         workinghours = self.workinghours_set.all().values("hours", "date", "reason")
         hour_sum = (participation.aggregate(Sum("hours"))["hours__sum"] or 0) + (
```

Comment:
Can you rename `participation` to `participations` and `hours` to duration or something like that?